src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,222 @@
+        List<Car> carList = new ArrayList<>();
+        String query = "SELECT model, manufacturer_id FROM cars c JOIN cars_drivers cd "
+                + "ON c.id = cd.car_id WHERE cd.driver_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, driverId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                carList.add(getCar(resultSet));
+            }
+            for (Car car : carList) {
+                car.setDrivers(getDriversByCar(car.getId()));
+            }
+            return carList;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get car from database with driver id: "
+                    + driverId, e);
+        }
+    }
+
+    private void insertDrivers(Car car) {
+        if (car.getDrivers() != null && !car.getDrivers().isEmpty()) {
+            String query = "INSERT INTO cars_drivers (car_id, driver_id) "
+                    + "VALUES (?, ?);";
+            try (Connection connection = ConnectionUtil.getConnection();
+                    PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+                preparedStatement.setLong(1, car.getId());
+                for (Driver driver : car.getDrivers()) {
+                    preparedStatement.setLong(2, driver.getId());
+                    preparedStatement.executeUpdate();
+                }
+            } catch (SQLException e) {
+                throw new DataProcessingException("Can`t add data in table cars_drivers", e);
+            }
+        }
+    }
+
+    private List<Driver> setDriversList(Long carId) {
+        List<Driver> listOfDrivers = new ArrayList<>();
+        String query = "SELECT id, name, license_number FROM drivers d "
+                + "JOIN cars_drivers cd ON d.id = cd.driver_id "
+                + "WHERE cd.driver_id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, carId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                listOfDrivers.add(getDrivers(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Cant set driver list for car with id: " + carId, e);
+        }
+        return listOfDrivers;
+    }
+
+    private Driver getDrivers(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getLong(1);
+        String name = resultSet.getString(2);
+        String licenseNumber = resultSet.getString(3);
+        return new Driver(id, name, licenseNumber);
+    }
+
+    private List<Driver> getDriversByCar(Long carId) {

```

Comment:
let's delete this method - you already have identical method 
private List<Driver> setDriversList(Long carId)